src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Rate/Rate.html

```diff
@@ -8,24 +12,30 @@
             'is-positive': rates(1) > rates(-1)}">
         {{rates(1) - rates(-1) | signum}}
     </span>
-    <a
-        href=""
-        class="rate-pro"
-        data-ng-class="{
-            'is-rate-button-active': myRate === 1,
-            'is-disabled': disabled || (!optionsPostPool.POST && optionsPostPool.loggedIn)
-        }"
-        data-ng-click="toggle(1)">
-        <i class="icon-vote-up" title="{{'TR__RATE_VOTE_UP' | translate}}"></i>
-    </a>
-    <a
-        href=""
-        class="rate-contra"
-        data-ng-class="{
-            'is-rate-button-active': myRate === -1,
-            'is-disabled': disabled || (!optionsPostPool.POST && optionsPostPool.loggedIn)
-        }"
-        data-ng-click="toggle(-1)">
-        <i class="icon-vote-down" title="{{'TR__RATE_VOTE_DOWN' | translate}}"></i>
-    </a>
+    <div class="rate-button-container">
```

Comment:
Until now the rate widget was `inline-block`ish. Can we use spans instead of divs?